doc/source/user_guide/groupby.rst

```diff
@@ -537,41 +574,84 @@ Another aggregation example is to compute the number of unique values of each gr
 
 .. note::
 
-   Aggregation functions **will not** return the groups that you are aggregating over
+   Aggregation functions **will not** operate on the groups that you are aggregating over
    if they are named *columns*, when ``as_index=True``, the default. The grouped columns will
    be the **indices** of the returned object.
 
    Passing ``as_index=False`` **will** return the groups that you are aggregating over, if they are
-   named *columns*.
+   named **indices** or *columns*.
```

Comment:
whether they are named indices or columns

Reply:
I _think_ this changes the meaning. To me, "whether" suggests that the cases that follow are exhaustive. That is not the case, e.g. the `by` argument in groupby can be a Python list. If the groups are not named indices or columns, they will not be added to the result when `as_index=False`.

Reply:
OK.